dev-libs/libsodium/libsodium-1.0.11.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A modern, easy-to-use software library for cryptographic standards"
+DESCRIPTION="Sodium is a portable, cross-compilable, installable, packageable \
+fork of NaCl, with a compatible API, and an extended API to improve usability \
+even further.
+Its goal is to provide all of the core operations needed to build higher-level \
+cryptographic tools.
+The design choices emphasize security, and \"magic constants\" have clear rationales.
+Despite the emphasis on high security, primitives are faster across-the-board \
+than most implementations of the NIST standards."
+HOMEPAGE="https://github.com/jedisct1/libsodium"
+COPYRIGHT="2013-2016  Frank Denis"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://download.libsodium.org/libsodium/releases/libsodium-$portVersion.tar.gz"
+CHECKSUM_SHA256="a14549db3c49f6ae2170cbbf4664bd48ace50681045e8dbea7c8d9fb96f9c765"
+PATCHES="libsodium-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libsodium${secondaryArchSuffix} = $portVersion
+	cmd:libsodium = $portVersion
+	lib:libsodium${secondaryArchSuffix} = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	"
+PROVIDES_devel="
+	libsodium${secondaryArchSuffix}_devel = $portVersion
+	devel:libsodium$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libsodium$secondaryArchSuffix == $portVersion base
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:automake
+	cmd:autoconf
+	cmd:libtool
+	cmd:make
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+  make && make check
```

Comment:
Hi @R167!

Could you please replace the **`make && make check`** by just **`make`** and add the following lines at the end of the recipe?
```

TEST()
{
	make check
}
```

Thanks a lot, by the way, for your taking part to GCI !